### DISCUSSION TITLE
Welcome to STM8 eForth Discussions! (read this first)

### DISCUSSION THREAD

**Discussion by TG9541** [TG9541/stm8ef#386] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with the [magic of a Forth console](https://github.com/TG9541/stm8ef/wiki/STM8-eForth-Programming) on an "approachable" chip, or also with [cheap Chinese STM8 boards](https://github.com/TG9541/stm8ef/wiki/STM8S-Value-Line-Gadgets).

![STM8S003](https://camo.githubusercontent.com/2d2a445ee074c06da54fd170d493e321b40af2d2f5d584030b531d1f1ffb0c74/68747470733a2f2f616530312e616c6963646e2e636f6d2f6b662f485442314b69794a4c58585858585853615858587136785846585858722f467265652d7368697070696e672d3130307063732d6c6f742d53544d3853303033463350362d53544d3853303033462d5453534f502d32302d49432e6a70675f323230783232302e6a7067)

**Comment by TG9541** [comment]
FYI: GitHub discussions went life today. I took the template as it is, but as of now I'm not certain if Issues isn't a better place for discussing stuff in this small community. Anyhow, feedback is welcome!

**Comment by uwu64** [comment]
Hey, not sure if this is a good place to write this, but I just want to comment about how grateful I am for efstm8. The resources have been absolutely massively helpful for getting started. Everything was really intuitive to set up. Me and my friends have had a ton of fun playing around with the interactivity of eForth. Thank you @TG9541 and contributors for your wonderful work!

**Reply by TG9541** [reply]
Thanks a bunch!
It's always good to get such feedback. Personally I find creating a low-complexity STM8 environment relaxing - it's different when there is a customer and you have to deliver, of course.
I've been following you guys over there in Oregon and I hope that you have great success with your ideas!
